docs/docs/mdx/writing-pages.md

```diff
@@ -114,6 +114,20 @@ export default ({ children }) => (
 > **Note**: the default export concept used in this code block is explained in more detail
 > in the docs below on [defining layouts](#defining-a-layout)
 
+## Combining frontmatter and imports
+
+If you would like to include frontmatter and import components, frontmatter needs to appear at the top of the file, imports can then follow after it:
+
+```mdx
+---
+title: "Building with Gatsby"
```

Comment:
I think the quotes can be removed from this one, too.

Reply:
```suggestion
title: Building with Gatsby
```